gnome-initial-setup/demo/eos-demo-video.c

```diff
@@ -51,6 +51,21 @@ on_demo_video_event (GtkWidget *widget,
   return FALSE;
 }
 
+static gboolean
+on_demo_video_became_active (GObject    *object,
+                             GParamSpec *pspec,
+                             gpointer    user_data)
+{
+  GtkWindow    *window = GTK_WINDOW (object);
+  GisDemoVideo *demo_video = GIS_DEMO_VIDEO (user_data);
+  GisDemoVideoPrivate *priv = gis_demo_video_get_instance_private (demo_video);
+
+  if (gtk_window_is_active (window))
+    gst_element_set_state (priv->player, GST_STATE_PLAYING);
+
+  return FALSE;
```

Comment:
Pretty sure that property notification signals return `void`

Reply:
> still surprised that gcc does not warn about this ....

Ah, that's actually because the return type of the function was already `gboolean`

Reply:
Fixed in any case